java/ql/src/semmle/code/java/frameworks/spring/SpringStringUtils.qll

```diff
@@ -0,0 +1,58 @@
+import java
+private import semmle.code.java.dataflow.ExternalFlow
+
+private class SpringStringUtilsModel extends SummaryModelCsv {
+  override predicate row(string row) {
+    row = [
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[0];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[1];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[1];Argument[0];taint",
+        "org.springframework.util;StringUtils;false;applyRelativePath;;;Argument[0];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;applyRelativePath;;;Argument[1];ReturnValue;taint",
```

Comment:
```suggestion
        "org.springframework.util;StringUtils;false;applyRelativePath;;;Argument[0..1];ReturnValue;taint",
```

and similarly any multiple args flowing to the same target

Reply:
Is there notation for 0 and 3?

Reply:
Fraid not